Assignment6-CRUD_FastAPI/config.py

```diff
@@ -0,0 +1,23 @@
+from dotenv import load_dotenv
+import os
+from pydantic import BaseModel
+
+load_dotenv()
+
+
+class Config(BaseModel):
+    db_user: str
+    db_password: str
+    db_server: str
+    db_name: str
+
+
+Config.db_user = os.environ.get('DB_USER')
```

Comment:
Consider adding default values to get() statements like these where you may not properly catch NULLs down the line

`Config.db_user = os.environ.get('DB_USER', '')`